features/cucumber.feature

```diff
@@ -147,6 +147,34 @@ Feature: The cucumber command
     But the output should not contain "other.feature"
 
 
+  Scenario: Running all cucumber features matching a given tag
+    Tags need to be passed differently than in vanilla cucumber where the syntax is --tags @tag.
+    With geordi you need to pass them like --tags=@tag to be parsed correctly.
```

Comment:
Instead of leaving this note to developers here, let's print a warning to the user if `cucumber` is invoked with `--tags` but without equals sign or without at sign.

Actually, doesn't cucumber.rb:103-104 enable `cucumber --tags @tag`, i.e. without equals sign?

Reply:
Well kind of, but that's only for args that are finally passed to the actual cucumber command.
In `lib/geordi/commands/cucumber.rb:55` we partition passed args according to whether they start with `-` or not. Args starting with `-` are considered command option, and those without are considered filepaths. If you don't pass tags with an `=` the tag itself will be considered a file.  

Then in `lib/geordi/cucumber.rb:13` we split the args at `=` and thus make use of the code you referenced.

Reply:
But your right, it would help to print this text as warning

Reply:
Ah I see. Could make the `cucumber` command use the argument partitioning from `Geordi::Cucumber#command_line_options` et al.? This would really be in the spirit of your MR.

If you do, we only need to warn about `--tags` without an @. Please remove that Cucumber scenario comment then.